### PR TITLE
Blog section

### PR DESCRIPTION
**Pull request by shivah12**
Still working on improving the blog view but added to the main and will add page for showing all blogs

### CODE REVIEW

**Comment by shivah12** [comment]
Screenshot_4-7-2024_13628_localhost

in the home section
Screenshot_4-7-2024_13614_localhost

**Comment by saurabh73** [comment]
@shivah12 There are conflicts , kindly resolve them first

**Comment by shivah12** [comment]
Okay sir

File: src/components/Projects.tsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+const SwiperNavButtons: React.FC<SwiperNavButtonsProps> = ({
+  handleButtonAction,
+  currentIndex,
+  maxIndex,
+}) => {
+  const swiper = useSwiper();
+
+  return (
+    <Flex gap={4} justifyContent={'center'} mt={4}>
+      <IconButton
```
Comment (ptabhishekutopia) [inline src/components/Projects.tsx:41]: we can create iconButton function and use it

File: static/admin/config.yml [hunk 2]
```diff
@@ -27,26 +27,44 @@ collections:
         }
       - { label: "Title", name: "title", widget: "string" }
       - { label: "Author", name: "author", widget: "string" }
-      - { label: "Image", name: "image", widget: "image"}
+      - { label: "Image", name: "image", widget: "image" }
       - {
           label: "Publish Date",
           name: "date",
           widget: "datetime",
           default: "",
```
Comment (ptabhishekutopia) [inline static/admin/config.yml:35]: no need of writing default, we can skip it

File: static/admin/config.yml [hunk 3]
```diff
@@ -67,50 +85,50 @@ collections:
...
-                    name: ctaButton,
-                    widget: list,
+                    label: "Button",
+                    name: "ctaButton",
+                    widget: "list",
                     fields:
                       [
-                        { label: Text, name: text, widget: string },
-                        { label: Link, name: link, widget: string },
+                        { label: "Text", name: "text", widget: "string" },
```
Comment (ptabhishekutopia) [inline static/admin/config.yml:98]: no need to add double inverted comma

File: src/components/Blog.tsx [hunk 4]
```diff
@@ -8,115 +8,96 @@ import {
...
-                      _hover={{
-                        transform: "scale(1.05)",
-                      }}
-                    />
-                  </Link>
-                </Box>
-                <BlogTags tags={["Engineering", "Product"]} marginTop="3" />
-                <Heading fontSize="xl" marginTop="2">
-                  <Link
-                    textDecoration="none"
```
Comment (ptabhishekutopia) [inline src/components/Blog.tsx:102]: no need of textdecoration in link , use only for hover

File: src/components/Blog.tsx
Comment (ptabhishekutopia) [inline src/components/Blog.tsx:1]: add dark mode

**Pull request closed without merging**